lib/uptime_web/router.ex

```diff
@@ -9,7 +9,7 @@ defmodule UptimeWeb.Router do
   @csp_script_src "script-src 'self'"
   @csp_img_src "img-src data: w3.org/svg/2000 'self'"
   @csp_font_src "font-src 'self' fonts.gstatic.com"
-  @csp_style_src "style-src 'self' fonts.googleapis.com"
+  @csp_style_src "style-src 'self' fonts.googleapis.com 'unsafe-inline'"
```

Comment:
I think we only want that for dev right?

Reply:
I was going to ask you about this. I need this in order to add style="" to a div that requires dynamic width %. How would this be handled in prod?